nservicebus/sqlserver/multiple-databases.md

```diff
@@ -1,21 +1,66 @@
 ---
-title: Multi-database support
-summary: How to configure SQL Server transport to use multiple instances of the database and route messages between them.
+title: SQL Server transport setup - default, multi-schema, multi-catalog and multi-instance
+summary: How to configure and manage SQL Server transport in default, multi-schema, multi-catalog and multi-instance modes
 tags:
 - SQL Server
 - Transports
 ---
 
-The SQL Server transport supports selecting, on per-endpoint basis, where the table queues should be created. The selection can be done on multiple levels:
+In the default SQL Server transport setup, the messages and other data are stored in multiple tables in a single database. The default schema is `dbo`, but it is possible to [specify a custom schema](/nservicebus/sqlserver/configuration.md#custom-database-schemas).
 
- * different schemas in a single database
- * different databases in a single SQL Server instance
- * different SQL Server instances
+In some environments it might be necessary to use a more advanced setup. The supported options are:
+ * ***default***: single schemas in a single database (Versions 1.x and higher)
+ * ***multi-schema***: multiple schemas in a single database (Versions 2.1 and higher)
+ * ***multi-catalog***: multiple databases in a single SQL Server instance (Versions 2.1 and higher)
```

Comment:
It should be (Versions 2.1 to 2.x) - we do not support it yet in V3 and wont' in 3.0